webpack.config.js

```diff
@@ -214,6 +214,7 @@ export default {
       },
       override: {
         'khroma@*': {licenseName: 'MIT'}, // https://github.com/fabiospampinato/khroma/pull/33
+        'htmx.org@1.9.10': {licenseName: 'BSD-2-Clause'}, // "BSD 2-Clause" -> "BSD-2-Clause"
```

Comment:
Should raise the issue upstream.

Reply:
They changed to 0BSD so next version should be valid

Reply:
Sounds like a new license type, but we can add it to the array below. The reason we have a allowlist of licenses is because SPDX does not have an expression for "all licenses", so we add all (acceptable) licenses to that array.

Reply:
I will do it when they release the next version